engine/classes/Elgg/I18n/Translator.php

```diff
@@ -219,7 +219,7 @@ public function loadTranslations($language = null) {
 		// Plugin translation have already been loaded for the default
 		// languages by ElggApplication::bootCore(), so there's no need
 		// to continue unless loading a specific language on-demand
-		if ($language) {
+		if ($language && !defined('PHPUNIT_ELGG_TESTING_APPLICATION')) {
```

Comment:
Are we abusing this global? We should eventually refactor to instead allow mocking plugin loader